test/loadTest/connections_stress_tests/README.md

```diff
@@ -0,0 +1,259 @@
+2016-12-20 10:25:09,546 | INFO |  -------- 13 -------- NOK -------- N/A -------- 5011 ---------- 0 -------- 5011 ----------
+2016-12-20 10:25:13,220 | INFO |  -------- 14 -------- NOK -------- N/A -------- 5011 ---------- 0 -------- 5011 ----------
+2016-12-20 10:25:16,861 | INFO |  -------- 15 -------- NOK -------- N/A -------- 5010 ---------- 0 -------- 5010 ----------
+...
+(Close the "telnet localhost 1026" session, releasing its connection)
+...
+2016-12-20 10:25:20,339 | INFO |  -------- 16 -------- OK -------- N/A -------- 5010 ---------- 0 -------- 5010 ----------
+2016-12-20 10:25:23,676 | INFO |  -------- 17 -------- OK -------- N/A -------- 5010 ---------- 0 -------- 5010 ----------
+2016-12-20 10:25:27,150 | INFO |  -------- 18 -------- OK -------- N/A -------- 5010 ---------- 0 -------- 5010 ----------
+```
+
+### Raising system limits for the test
+
+In the case of running CB as foreground process:
+
+* If you are not using CentOS 6.x, probably just using `ulimit -u <new limit>` and `ulimit -n <new limit>`
+  to change respectively the max number of processes and the max number of open file descriptors will work.
+* If you are using CentOS 6.x, edit the following file (as root): `/etc/security/limits.d/90-nproc.conf`
+  as shown below, then re-login (more detail in [this reference](https://bugzilla.redhat.com/show_bug.cgi?id=919793)):
+
+```
+*          soft    nproc     10240
+*          soft    nofile    10240
+*          hard    nproc     10240
+*          hard    nofile    10240
+```
+
+In the case of running CB as a service in CentOS:
+
+* The contextBroker process is run by the `orion` user with default configuration. The above configuration
+  in /etc/security/limits.d/90-nproc.conf should suffice but anyway it is advisable to check the limits
+  with `cat /proc/<pid>/limits` once the service has been started.
+* You have to run RPyC as `orion` user. Otherwise, the connection inspection done by connections_stress_tests.py
+  will fail (rpyc_classic.py is the actual process that looks for CB process connections so it has to be run by the
+  same user that runs CB in order to have enough privilege level).
+
+```
+$ sudo su orion -c rpyc_classic.py
+INFO:SLAVE/18812:server started on [0.0.0.0]:18812
+```
+
+### Detailed `connections_stress_tests.py` usage
+
+       Parameters:                                                                                                  
+          -host=<host>         : CB host (OPTIONAL) (default: localhost).                                           
+          -u                   : show this usage (OPTIONAL).                                                        
+          -v                   : verbose with all responses (OPTIONAL) (default: False). 
+          -noEstablished       : is used to ignore the established connections (OPTIONAL) (default: False).                         
```

Comment:
Potential change from `-noEstablished` to `-noConnectionInfo`.

In addition, the message should be aligned with the one actually returned by the script (currently it isn't).